### PR TITLE
Release manager to add/remove people from release-roles channel

### PR DESCRIPTION
**Pull request by DanielMurphy22**
**Description of work.**

Remind Release Manager to add/remove people to/from the #release-roles Slack Channel

**To test:**

Check that comment is correctly worded and you agree this is part of the Release Manager's role.

*There is no associated issue.*

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?
- Are the release notes saved in a separate file, using Issue or PR number for file name and in the correct location?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.

### CODE REVIEW

File: dev-docs/source/ReleaseChecklist.rst [hunk 1]
```diff
@@ -191,6 +191,7 @@ all of the Local Project Managers.
 1 week before Code Freeze
 #########################
 
+*  Check that all people with release roles are added to the *\#release-roles* Slack channel and others are removed.
```
Comment (ConorMFinn) [inline dev-docs/source/ReleaseChecklist.rst:194]: This is peak nitpick, but can we keep the paragraphs about the same length per line?
```suggestion
*  Check that all people with release roles are added to the *\#release-roles* 
   Slack channel and others are removed.
```

**Review by cailafinn — CHANGES_REQUESTED** [review]
This is such a nit, I'm so sorry

File: dev-docs/source/ReleaseChecklist.rst (hunk 1 above)
Comment (DanielMurphy22) [reply dev-docs/source/ReleaseChecklist.rst:194]: Peak!

Comment (DanielMurphy22) [reply dev-docs/source/ReleaseChecklist.rst]: I think my way formatted fine and wasn't beyond the line length according to pycharm! I've made the change anyway!

Comment (ConorMFinn) [reply dev-docs/source/ReleaseChecklist.rst]: Oh yeah, it didn't affect anything on the actual docs, just so it was the same as the surrounding code. That's why I almost felt bad mentioning it 😅

**Review by cailafinn — APPROVED** [review]
LGTM! 👍

**Pull request merged**